maltego_trx/server.py

```diff
@@ -1,29 +1,32 @@
 import logging
+from xml.etree.ElementTree import Element, SubElement
 
 from flask import Flask, request
+
 from maltego_trx.maltego import MaltegoMsg
 from .registry import mapping
+from .utils import serialize_xml
 
 log = logging.getLogger("maltego.server")
 logging.basicConfig(level=logging.DEBUG)
 
-
 URL_TEMPLATE = '/run/<transform_name>/'
 URL_TEMPLATE_NO_SLASH = '/run/<transform_name>'
 
 
+def build_exception_message(msg="An exception occurred with the transform. Check the logs for more details."):
```

Comment:
This serialization logic already exists on `MaltegoTransform`.

Could have also done something along the lines of:
```
transform_run = MaltegoTransform()
transform_run.addUIMessage(msg, "PartialError")
return transform_run.returnOutput()
```